test/core.vwtest.json

```diff
@@ -5895,5 +5895,16 @@
     "input_files": [
       "train-sets/las_100_actions.txt"
     ]
+  },
+  {
+    "id": 456,
+    "desc": "Spin off epsilon decay model to lower bitsize to cb_explore with explore eval",
+    "vw_command": "--cb_explore_adf -d train-sets/automl_spin_off.txt --noconstant -b 18 --predict_only_model --epsilon_decay --model_count 2 --explore_eval",
```

Comment:
Is this meant to be in this PR?

Reply:
I think so

Reply:
It's not related to metrics or explore eval though